supervision/classification/core.py

```diff
@@ -40,6 +42,9 @@ def __post_init__(self) -> None:
         _validate_confidence(self.confidence, n)
 
     @classmethod
+    @deprecated(
+        "This method is deprecated and removed in 0.15.0 release. Use sv.Classifications.from_ultralytics() instead."
```

Comment:
Let's make that `0.16.0`. We should give people at least 2 full release cycles to migrate. That change will be released with `0.14.0`.

Reply:
Done!